### DISCUSSION TITLE
Quarto Website Consistent Formats

### DISCUSSION THREAD

**Discussion by ivanuricardo** [quarto-dev/quarto-cli#7233] [Q&A]
### Description

Hello! I am currently developing a website using quarto and had a question regarding formatting. First, I wanted to remove the sidebar and margins on my home page such that I only have an image at the top with some information following. My index.qmd looks like this
```
---
image: assets/backpacking.jpg
about:
  template: marquee
  image-width: 1500px
format:
  html:
    margin-top: -1em
    grid:
      body-width: 2800px
      sidebar-width: 0px
      margin-width: 0px
      gutter-width: 0px
---
:::: {.columns}

::: {.column width="2%"}

:::

::: {.column width="13%"}
![](assets/bio-photo-modified.png){width=60%}

...
:::

::: {.column width="2%"}

:::

::: {.column width="80%"}
# Welcome
...
:::

::: {.column width="3%"}

:::

::::

```

However, when making my blogs page, I want to change the format such that the sidebar comes back. So my blog.qmd should have a different grid format. However, when I change it, the format changes for both my blog page and my home page. Is there any way of having different formats for different pages?
Thanks!

**Comment by mcanouil** [comment]
It would be more helpful to share a complete (and small) project rather than only one Quarto document which actually does not represent the reality of your use case.
